src/main/io/gps.c

```diff
@@ -263,37 +324,35 @@ typedef struct ubxMessage_s {
 } __attribute__((packed)) ubxMessage_t;
 
 typedef enum {
-    UBLOX_INITIALIZE,
-    UBLOX_MSG_VGS,      //  1. VGS: Course over ground and Ground speed
-    UBLOX_MSG_GSV,      //  2. GSV: GNSS Satellites in View
-    UBLOX_MSG_GLL,      //  3. GLL: Latitude and longitude, with time of position fix and status
-    UBLOX_MSG_GGA,      //  4. GGA: Global positioning system fix data
-    UBLOX_MSG_GSA,      //  5. GSA: GNSS DOP and Active Satellites
-    UBLOX_MSG_RMC,      //  6. RMC: Recommended Minimum data
-    UBLOX_MSG_RATE,     //  7. set PVT or SOL MSG rate
-    UBLOX_MSG_POSLLH,   //  8. SOL: set POSLLH MSG rate
-    UBLOX_MSG_STATUS,   //  9: set STATUS MSG rate
-    UBLOX_MSG_VELNED,   // 10. SOL: set VELNED MSG rate
-    UBLOX_MSG_SATINFO,  // 11. MSG_SAT or MSG_SVINFO if not ubloxUseSAT
-    UBLOX_MSG_DOP,      // 12. MSG_DOP 
-    UBLOX_SET_NAV_RATE, // 13. set GPS sample rate
-    UBLOX_SET_SBAS,     // 14. Sets SBAS
-    UBLOX_SET_PMS,      // 15. Sets Power Mode
-    UBLOX_MSG_CFG_GNSS  // 16. For not SBAS or GALILEO otherwise GPS_STATE_REVEIVING_DATA
+    UBLOX_DETECT_UNIT,      //  0
+    UBLOX_INITIALIZE,       //  1. 
+    UBLOX_ACQUIRE_MODEL,    //  2
+//    UBLOX_CFG_ANA,          //  3. ANA: if M10, enable autonomous mode : temporarily disabled.
+    UBLOX_MSG_DISABLE_NMEA, //  4. Disable NMEA
+    UBLOX_MSG_VGS,          //  5. VGS: Course over ground and Ground speed
+    UBLOX_MSG_GSV,          //  6. GSV: GNSS Satellites in View
+    UBLOX_MSG_GLL,          //  7. GLL: Latitude and longitude, with time of position fix and status
+    UBLOX_MSG_GGA,          //  8. GGA: Global positioning system fix data
+    UBLOX_MSG_GSA,          //  9. GSA: GNSS DOP and Active Satellites
+    UBLOX_MSG_RMC,          // 10. RMC: Recommended Minimum data
+    UBLOX_SET_SBAS,         // 11. Sets SBAS
+    UBLOX_SET_PMS,          // 12. Sets Power Mode
+    UBLOX_MSG_NAV_PVT,      // 13. set NAV-PVT rate
+    UBLOX_MSG_SOL,          // 14. set SOL MSG rate
+    UBLOX_MSG_POSLLH,       // 15. set POSLLH MSG rate
+    UBLOX_MSG_STATUS,       // 16: set STATUS MSG rate
+    UBLOX_MSG_VELNED,       // 17. set VELNED MSG rate
+    UBLOX_MSG_DOP,          // 18. MSG_NAV_DOP
+    UBLOX_SAT_INFO,         // 19. MSG_NAV_SAT message
+    UBLOX_SET_NAV_RATE,     // 20. set to user requested GPS sample rate
+    UBLOX_MSG_CFG_GNSS      // 21. For not SBAS or GALILEO otherwise GPS_STATE_REVEIVING_DATA
 } ubloxStatePosition_e;
 
-#endif // USE_GPS_UBLOX
+baudRate_e initBaudRateIndex;
+size_t initBaudRateCycleCount;
+static void ubloxSendClassMessage(ubxProtocolBytes_e class_id, ubxProtocolBytes_e msg_id, uint16_t length);
 
-typedef enum {
-    GPS_STATE_UNKNOWN,
-    GPS_STATE_INITIALIZING,
-    GPS_STATE_INITIALIZED,
-    GPS_STATE_CHANGE_BAUD,
-    GPS_STATE_CONFIGURE,
-    GPS_STATE_RECEIVING_DATA,
-    GPS_STATE_LOST_COMMUNICATION,
-    GPS_STATE_COUNT
-} gpsState_e;
```

Comment:
Removing this from `gps.c` is not needed

Reply:
This enum is used in cli now